feat(HTTP): Refresh 4XX response pages (#34620)
mdn/content: files/en-us/web/http/status/406/index.md

{{HTTPSidebar}}

> The HTTP **`406 Not Acceptable`** [client error response](/en-US/docs/Web/HTTP/Status#client_error_responses) status code indicates that the server could not produce a response matching the list of acceptable values defined in the request's [proactive content negotiation](/en-US/docs/Web/HTTP/Content_negotiation#server-driven_content_negotiation) headers, and that the server was unwilling to supply a default representation.

Reviewer: ```suggestion
The HTTP **`406 Not Acceptable`** [client error response](/en-US/docs/Web/HTTP/Status#client_error_responses) status code indicates that the server could not produce a response matching the list of acceptable values defined in the request's [proactive content negotiation](/en-US/docs/Web/HTTP/Content_negotiation#server-driven_content_negotiation) headers and that the server was unwilling to supply a default representation.
```

{{HTTPSidebar}}

The HTTP **`406 Not Acceptable`** status response code is part of the `400`-`499` class of [client error responses](/en-US/docs/Web/HTTP/Status#client_error_responses) and indicates that the server cannot produce a response matching the list of acceptable values defined in the request's proactive [content negotiation](/en-US/docs/Web/HTTP/Content_negotiation) headers, and that the server is unwilling to supply a default representation.

Proactive content negotiation headers include:

- {{HTTPHeader("Accept")}}
- {{HTTPHeader("Accept-Encoding")}}
- {{HTTPHeader("Accept-Language")}}

> A server may return responses which are not in the accept headers sent in the client request, and this may be preferable to sending a 406 response in some cases.

Reviewer: I did not understand this sentence. This is what I thought you meant, but I might be wrong:

```suggestion
A server may return responses that differ from the accept headers sent in the client request. In some cases, this may be preferable to sending a 406 response.
```

Author: slight reword in https://github.com/mdn/content/pull/34620/commits/5ec35dbdd45b72e904277ceab6f6106a745a6c10
